client/src/views/NextLaunch.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <v-container>
+      <h5 class="text-h5 text-md-h4 font-weight-bold text-center my-10 secondary">
+        Codescalers' CloudForStudents New Launch is soon to come!
```

Comment:
Can we change it to `We have reached the end of this chapter, but the story isn't over. Stay tuned for the next exciting lanuch`